.github/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Sonobuoy version: 
+- Kubernetes version: (use `kubectl version`):
+- Kubernetes installer & version:
```

Comment:
maybe this could be: Kubernetes installer & installer version (e.g. kops, kubeadm, kubespray)